packages/table/src/Table/TableWithVS.story.tsx

```diff
@@ -49,52 +50,54 @@ const meta: StoryMetaType<typeof Table> = {
 export default meta;
 
 const virtualScrollingContainerHeight = css`
-  height: 500px;
+  /* height: 500px; */
```

Comment:
these both feel like magic numbers?

Reply:
i guess its in the story file

Reply:
Yeah, they're arbitrary but it's the story file so it's not a huge deal